cli/execute/execute.go

```diff
@@ -149,6 +149,8 @@ func execute(cmdArgs []string) error {
 		return err
 	}
 	log.Debugf("Uploaded inputs in %s", time.Since(stageStart))
+	actionStr, _ := digest.ResourceNameFromProto(arn).DownloadString()
```

Comment:
Probably want to check this error, just in case, no?

Reply:
Done